okhttp-tests/src/test/java/okhttp3/internal/http2/HttpOverHttp2Test.java

```diff
@@ -1266,6 +1263,10 @@ private int countFrames(List<String> logs, String message) {
       }
     });
 
+    // Force a fresh idle connections
+    client.connectionPool().evictAll();
+    int activeConnections = client.connectionPool().connectionCount();
```

Comment:
https://github.com/square/okhttp/issues/4030

Reply:
Please separate this out into a separate PR.

Reply:
@yschimke If this bug is confirmed, then it will be necessary to correct not only this test.
The problem is that evictAll() does not close the all active connections and this test is occasionally fail as it was always expected 1.
Without this change, mvn clean verify fail with an error in this test.
If I remove this change, then the build may not be stable.

Reply:
It's not related to this desired API change so it should be separate.

Reply:
![image](https://user-images.githubusercontent.com/15269131/40589128-b99c3790-61f0-11e8-8be8-2ec258703e82.png)

Reply:
Yes. It is unrelated and should be a separate PR.